docs/usage/quickstart.rst

```diff
@@ -5,33 +5,101 @@
    Todo
    usernames
 
+
 Quickstart Guide
 ================
-To quickly set up a testing cluster using MetalK8s_, you need 3 machines
-running CentOS_ 7.4 to which you have SSH access (these can be VMs). Each
-machine acting as a Kubernetes_ node (all of them, in this example) also
-need to have at least one disk available to provision storage volumes.
-
-.. todo:: Give some sizing examples
+This guide describes how to set up a MetalK8s_ test cluster. It offers general
+requirements and describes sizing, configuration, and deployment. With respect
+to installation procedures, the only significant difference between a test
+cluster and a full production environment is the amount of resources required
+to implement the design.
```

Comment:
Given this, should we call it a 'test' cluster in the first sentence?

Reply:
meaning you'd prefer not calling it a 'test' cluster?

Reply:
Indeed. If this is the installation guide, I'd assume it's meant to install clusters, for testing purposes or other.